src/components/BottomBar/index.js

```diff
@@ -29,41 +38,62 @@ class BottomBar extends React.PureComponent {
   }
 
   render() {
-    const { syncBlockNum, syncBlockTime } = this.props;
+    const { classes, syncBlockNum, syncBlockTime } = this.props;
+
+    return (
+      syncBlockTime && syncBlockTime ?
+        <Paper className={classes.bottomBarWrapper}>
+          <Grid container>
+            {this.renderNetworkConnection()}
+            {this.renderBlockInfo()}
+          </Grid>
+        </Paper> : null
```

Comment:
do you think we should still show this bar even if the we cant get syncBlockTime/num? and maybe show error text: `Error getting block info`?

Also, the network connection text should be visible always

Reply:
Let's hide the block number if it's not loaded. It's not nice to display error to users directly. And there is the loading phase so...